atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2013,8 +2007,8 @@ void WebContents::ShowCertificate() {
       web_contents()->GetController().GetVisibleEntry()->GetSSL().certificate;
   if (!certificate)
     return;
-  web_contents()->GetDelegate()->ShowCertificateViewerInDevTools(
-      web_contents(), certificate);
+  // web_contents()->GetDelegate()->ShowCertificateViewerInDevTools(
```

Comment:
wip?

Reply:
Yes. That's one of the two patches that were pending https://github.com/brave/muon/pull/322/commits/519e336932ca3a5a4e378ba8fab4c17170ed056a

Reply:
have a fix in progress for this as well